### ISSUE TITLE
Chronometer alarm usually continues playing continually until stopped manually

### ISSUE THREAD

**Issue by PaddlesAU** [raiguard/ModernGadgets#268]
I've found an issue where _most_ of the time, Chronometer alarms that have activated will keep playing continuously, until they're stopped manually, rather than stopping when they reach the expected duration.

The first time the alarm runs after Rainmeter starts (or the first time it runs after the Chronometer Settings widget is opened... which makes testing difficult because I can only test every 24 hours), it usually works as expected - the problem seems to happen more consistently on subsequent runs. Refreshing the skin manually doesn't seem to make a difference.

I've tried watching the skin measures under Rainmeter/Open Log/Skins, but I can't see anything odd to indicate why it isn't happening. The MeasureAlarm[1|2]Loop loop seems to go through the cycle as expected, so I don't know why IfTrueAction4 isn't working. (Edit: for the first time tonight I had this open prior to the alarm starting, and this time MeasureAlarm1Loop stayed disabled when the alarm started, but I've never seen this before. 

Chronometer.ini and ChronometerSettings.inc attached.
[Chronometer.ini](https://github.com/raiguard/ModernGadgets/files/7554380/Chronometer.ini.txt)
[ChronometerSettings.inc](https://github.com/raiguard/ModernGadgets/files/7554383/ChronometerSettings.inc.txt)

**Comment by PaddlesAU** [comment]
Update: behaviour seems to have changed, I'm not sure if this is due to updates in Rainmeter itself? It now looks like on the first run, the loop runs fine - but on subsequent runs, MeasureAlarm[1|2]Loop is stuck on zero and so never reaches the counter. (I've been gradually learning more about logging etc. to try and solve it.)

I've tried inserting a command to reset the measure via MeasureAlarm[1|2]Diff - e.g. in MeasureAlarm1Diff, I've added a Reset command into IfTrueAction (bolded below):
New version:
[MeasureAlarm1Diff]
Measure=Uptime
Format=%4!i!d %3!i!h %2!i!m %1!i!s
SecondsValue=([MeasureAlarm1Time:TimeStamp] - [MeasureLocalTimeOfDay:TimeStamp])
DynamicVariables=1
Disabled=(#enableAlarm1# = 0)
IfCondition=MeasureAlarm1Diff = 0
IfTrueAction=**[!CommandMeasure MeasureAlarm1Loop Reset]**[!EnableMeasure MeasureAlarm1Loop]

This seems to have the desired effect - the loop is running as expected and the alarm is stopping at the expected time.
